.gitmodules

```diff
@@ -30,3 +30,9 @@
 [submodule "include/Detours"]
 	path = include/detours
 	url = https://github.com/microsoft/Detours/
+[submodule "include/microsoft-ui-uiautomation"]
+	path = include/microsoft-ui-uiautomation
+	url = https://github.com/michaeldcurran/microsoft-ui-uiautomation
```

Comment:
Are you intending to refer to your fork here, and not microsoft/microsoft-ui-uiautomation?

Reply:
Yes, there is still one PR I'm waiting to be merged to upstream.